packages/augur-api/src/ethereum/types.ts

```diff
@@ -1,34 +1,35 @@
 export interface Log {
-  blockNumber?: number;
-  blockHash?: string;
-  transactionIndex?: number;
-  removed?: boolean;
-  transactionLogIndex?: number;
+  blockNumber: number;
+  blockHash: string;
+  transactionIndex: number;
+  removed: boolean;
+  transactionLogIndex: number;
   address: string;
   data: string;
   topics: Array<string>;
-  transactionHash?: string;
-  logIndex?: number;
+  transactionHash: string;
+  logIndex: number;
 }
 
 export interface LogValues {
   [paramName: string]: any;
 }
 
 export interface ParsedLog {
-  blockNumber?: number;
-  blockHash?: string;
-  transactionIndex?: number;
-  removed?: boolean;
-  transactionLogIndex?: number;
-  transactionHash?: string;
-  logIndex?: number;
+  blockNumber: number;
```

Comment:
It seems odd that these were made optional. I suppose as long as it still runs as expected, it's good to change them.

Reply:
I think they were made required on this branch.

Reply:
Right, I just mean I don't know why they were originally made optional. Seems like they should always be expected to be present.

Reply:
Ah. Yeah, hopefully marking these required makes life simpler.